tests/Evaluator/Strategies/test_full_mixed_strategies_evaluator.py

```diff
@@ -52,7 +52,7 @@ class TestFullMixedStrategiesEvaluator(AbstractStrategyTest):
 
     @staticmethod
     async def test_slow_uptrend(strategy_tester):
-        await strategy_tester.run_test_slow_uptrend(-6, -2)
+        await strategy_tester.run_test_slow_uptrend(7, -2)
```

Comment:
Why this value has been modified ?

Reply:
because of the solving of  https://github.com/Drakkar-Software/OctoBot/issues/834
it was a bug with reference market to compute profitability

Reply:
Oh ok nice !